src/front/ticketer/lib/backend_communication/model/event_status.dart

```diff
@@ -1,13 +1,43 @@
-// ignore_for_file: constant_identifier_names
-
 enum EventStatus {
-  Unverified,
-  Verified,
-  Published,
-  Opened,
-  Closed,
-  Finished,
-  Cancelled,
-  Held,
-  Recalled,
+  Unverified(0),
+  Verified(1),
+  Published(2),
+  Opened(3),
+  Closed(4),
+  Finished(5),
+  Cancelled(6),
+  Held(7),
+  Recalled(8);
+
+  const EventStatus(this.value);
+  final int? value;
+
+  static EventStatus getByCode(int i) {
+  return EventStatus.values.firstWhere((x) => x.value == i);
+  }
```

Comment:
```suggestion
  static EventStatus getByCode(int i) =>
      EventStatus.values.firstWhere((x) => x.value == i);
```

Reply:
This function doesn't have to have a body

Reply:
Addressed